internal/cmd/ceapi.go

```diff
@@ -81,3 +82,94 @@ func findEnv(ctx context.Context, client *coder.Client, envName, userEmail strin
 		clog.Tipf("run \"coder envs ls\" to view your environments"),
 	)
 }
+
+func findImg(ctx context.Context, client *coder.Client, email, imgName string) (*coder.Image, error) {
+	userImgs, err := lookupUserImgs(ctx, client, email, imgName)
+	if err != nil {
+		return nil, err
+	}
+
+	numImgs := len(userImgs)
+
+	if numImgs == 0 {
+		return nil, xerrors.New("image not found - did you forget to import this image?")
+	}
+
+	if numImgs > 1 {
+		var lines []string
+
+		for _, img := range userImgs {
+			lines = append(lines, img.Repository)
+		}
+
+		clog.LogInfo(
+			fmt.Sprintf("Found %d possible matches for %q.", numImgs, imgName),
+			clog.Tipf("Did you mean?\n\t%s", strings.Join(lines, "\n\t")),
+		)
+
+		return nil, xerrors.New("please refine your search")
+	}
+	return userImgs[0], nil
+}
+
+func getImgs(ctx context.Context, client *coder.Client, userEmail string) ([]*coder.Image, error) {
+	u, err := client.UserByEmail(ctx, userEmail)
+	if err != nil {
+		return nil, err
+	}
+
+	orgs, err := client.Organizations(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	orgs = lookupUserOrgs(u, orgs)
+	var importedImgs []*coder.Image
+
+	// Get all of the imported images for all of the orgs the user belongs to.
+	for _, org := range orgs {
+		imgs, err := client.GetOrganizationImages(ctx, org.ID)
+		if err != nil {
+			return nil, err
+		}
+		importedImgs = append(importedImgs, imgs...)
+	}
+	return importedImgs, nil
+}
+
+// returns all images that contain imgName as a substring and have been imported by the user.
+func lookupUserImgs(ctx context.Context, client *coder.Client, email, imgName string) ([]*coder.Image, error) {
```

Comment:
This is quite confusing behavior for this function. It should either return a single image if there is an exact match or an err. To include the "help" information in the error use `clog.Fatal`.